test/init.js

```diff
@@ -111,7 +111,7 @@ describe('#init', () => {
 
   it('should set default values when not specified', () => {
     statsd = createHotShotsClient({}, clientType);
-    assert.equal(statsd.host, 'localhost');
```

Comment:
Tests are failing now- looks like this needs to now change to undefined

Reply:
Thanks!